src/server/scripts/EasternKingdoms/ScarletEnclave/chapter1.cpp

```diff
@@ -1234,7 +1234,48 @@ class npc_scarlet_miner : public CreatureScript
         }
 };
 
-// npc 28912 quest 17217 boss 29001 mob 29007 go 191092
+enum StableMasterRepo
+{
+    SPELL_REPO = 52265
+};
+
+class spell_stable_master_repo : public SpellScriptLoader
+{
+public:
+    spell_stable_master_repo() : SpellScriptLoader("spell_stable_master_repo") { }
+
+    class spell_stable_master_repo_AuraScript : public AuraScript
+    {
+        PrepareAuraScript(spell_stable_master_repo_AuraScript);
+
+        bool Validate(SpellInfo const* /*spellInfo*/) override
+        {
+            return ValidateSpellInfo({ SPELL_REPO });
```

Comment:
You don't need this. I believe it's only for spells that you cast in functions below.

Reply:
the id which the spellscript is binded into is validated on loading

Reply:
Nice, thanks! Copied that from another script, thought it was needed.